tests/Commons.Persistence.EntityFramework.Tests/Generic/EntityDaoTests.cs

```diff
@@ -73,5 +81,74 @@ public async Task TestLoadByBusinessIdAsync()
                 actualEntity.Should().BeEquivalentTo(expectedEntity);
             }
         }
+
+        [Test]
+        public void TestFindByBusinessIds()
+        {
+            Guid[] idsToFind = new Guid[2];
+            List<EntityWithStringKey> expectedEntities = new List<EntityWithStringKey>();
+
+            using (TestDbContext context = new TestDbContext(contextOptions))
+            {
+                EntityDao<EntityWithStringKey, string> entityDao = new EntityDao<EntityWithStringKey, string>(context);
+                EntityWithStringKey entity1 = new EntityWithStringKey("Erste Id");
+                EntityWithStringKey entity2 = new EntityWithStringKey("Zweite Id");
+                EntityWithStringKey entity3 = new EntityWithStringKey("Dritte Id");
+                EntityWithStringKey entity4 = new EntityWithStringKey("Vierte Id");
+                List<EntityWithStringKey> entities = new List<EntityWithStringKey>() { entity1, entity2, entity3, entity4 };
+                entityDao.Save(entities);
+                idsToFind[0] = entity1.BusinessId;
+                idsToFind[1] = entity4.BusinessId;
+                expectedEntities.Add(entity1);
+                expectedEntities.Add(entity4);
+                context.SaveChanges();
+
+            }
```

Comment:
```
            Guid[] idsToFind = new Guid[2];
            List<EntityWithStringKey> expectedEntities = new List<EntityWithStringKey>();

            using (TestDbContext context = new TestDbContext(contextOptions))
            {
                EntityDao<EntityWithStringKey, string> entityDao = new EntityDao<EntityWithStringKey, string>(context);
                EntityWithStringKey entity1 = new EntityWithStringKey("Erste Id");
                EntityWithStringKey entity2 = new EntityWithStringKey("Zweite Id");
                EntityWithStringKey entity3 = new EntityWithStringKey("Dritte Id");
                EntityWithStringKey entity4 = new EntityWithStringKey("Vierte Id");
                List<EntityWithStringKey> entities = new List<EntityWithStringKey>() { entity1, entity2, entity3, entity4 };
                entityDao.Save(entities);
                idsToFind[0] = entity1.BusinessId;
                idsToFind[1] = entity4.BusinessId;
                expectedEntities.Add(entity1);
                expectedEntities.Add(entity4);
                context.SaveChanges();

            }
```

Ist doppeltert Code, der auch  TestFindByBusinessIdsAsync() benutzt werden kann, deher würde ich vorschlagen das in eine private Methode zu extrahieren.